config/software/libyaml-windows.rb

```diff
@@ -27,8 +27,8 @@
 name "libyaml-windows"
 default_version "0.1.6"
 
-source :url => "http://packages.openknapsack.org/libyaml/libyaml-0.1.6-x86-windows.tar.lzma",
-       :md5 => "8bb5d8e43cf18ec48b4751bdd0111c84",
+source :url => "https://dl.bintray.com/oneclick/OpenKnapsack/x86/libyaml-0.1.7-x86-widows.tar.lzma",
```

Comment:
We're missing a `:` in the URL : `https://dl.bintray.com/oneclick/OpenKnapsack/x86/:libyaml-0.1.7-x86-windows.tar.lzma`

Reply:
we should be using `#{version}` instead of hardcoding the version, and update `default_version` above